tracer/src/Datadog.Trace/Propagators/DatadogContextPropagator.cs

```diff
@@ -74,7 +76,9 @@ private DatadogContextPropagator()
             var origin = ParseUtility.ParseString(carrier, carrierGetter, HttpHeaderNames.Origin);
             var propagatedTraceTags = ParseUtility.ParseString(carrier, carrierGetter, HttpHeaderNames.PropagatedTags);
 
-            var traceTags = TagPropagation.ParseHeader(propagatedTraceTags);
+            // HACK: fix this so we don't parse "x-datadog-tags" twice
```

Comment:
I don't see "x-datadog-tags" being parsed twice, is this comment outdated?

Reply:
It is outdated! I fixed the need to parse twice by parsing earlier in #3872.

Reply:
Remove obsolete comment, thanks.